charts/tidb-operator/templates/controller-manager-rbac.yaml

```diff
@@ -56,6 +56,12 @@ rules:
   - restores
   - restores/finalizers
   verbs: ["*"]
+- apiGroups:
+  - apps.pingcap.com
+  resources:
+  - statefulsets
+  verbs:
+  - '*'
```

Comment:
Should this be configured only when the advance sts feature is turned on, keeping the principle of least privilege?

Reply:
done